app/data/DataStores.scala

```diff
@@ -61,6 +63,7 @@ trait UnpackedDataStores {
   val previewDataStore: PreviewDataStore = stores.preview
   val publishedDataStore: PublishedDataStore = stores.published
   val auditDataStore: AuditDataStore = stores.audit
+  val plutoDataStore: PlutoDataStore = stores.pluto
```

Comment:
These guys are only here for compatibility with older code. I'd just call `stores.pluto` instead